docs/topics/run-code-snippets.md

```diff
@@ -1,14 +1,14 @@
 [//]: # (title: Run code snippets – tutorial)
 
-There may be times when you need to quickly write and execute code outside of a project or application. 
-This may be useful, for example, when learning Kotlin or evaluating expressions.
-Let's have a look at three handy ways we can use to run Kotlin code quickly:
+Kotlin code is typically organized into projects with which you work in an IDE, a text editor, or other tools. However,
+if you want to quickly check how a function works or find an expression's value, there's no need to create a new project
+and build it: check out three handy ways to run Kotlin code instantly in different environments:
 
-* [Scratches](#scratches-and-worksheets) let us write and run code in a temporary file outside of our project in the IDE.
-* [Worksheets](#scratches-and-worksheets) are like scratches, but they reside within projects.
-* [REPL](#repl) (_Read-Eval-Print-Loop_) runs code in an interactive console.     
+* [Scratches and worksheets](#intellij-idea-scratches-and-worksheets) in IntelliJ IDEA
+* [Kotlin Playground](#browser-kotlin-playground) in the browser
+* [ki shell](#command-line-ki-shell) in the command line
 
-## Scratches and worksheets
+## IntelliJ IDEA: Scratches and worksheets
```

Comment:
AFAIK AndroidStudio also supports scratches. Shouldn’t we mention it?

Reply:
Yeah, it does. Let's change this to `IDE` and mention Android Studio in this section.